tests/manage/test_create_pvc_random_storage_class.py

```diff
@@ -50,17 +46,20 @@ def teardown():
     """
     global RBD_SECRET_OBJ, CEPHFS_SECRET_OBJ
     log.info("Deleting PVC")
-    assert pvc.delete_all_pvcs()
+    pvc_names = [pvcs for pvcs in PVC_OBJS]
+    assert pvc.delete_pvcs(pvc_names)
     log.info("Deleting CEPH BLOCK POOL")
-    assert helpers.delete_cephblockpool()
+    for pool in POOL_OBJS:
```

Comment:
so,
you have a function to delete pvcs and not to delete cephvlockpools, let's try to be consistent

Reply:
done